packages/secure_storage/amplify_secure_storage/README.md

```diff
@@ -2,12 +2,6 @@
 
 A package for storing secrets, intended for use in Amplify libraries.
 
-## Developer Preview
```

Comment:
Why are we removing this?

Reply:
It didn't seem to make sense there. Same reason it's not included in worker bee or smithy packages.

Reply:
Ok, sounds good.